packages/ckeditor5-image/tests/imagetextalternative/imagetextalternativeui.js

```diff
@@ -110,6 +110,16 @@ describe( 'ImageTextAlternativeUI', () => {
 
 			sinon.assert.callOrder( disableCssTransitionsSpy, addSpy, selectSpy, enableCssTransitionsSpy );
 		} );
+
+		it( 'has isOn bound to command\'s value', () => {
+			const button = editor.ui.componentFactory.create( 'imageTextAlternative' );
```

Comment:
It is not necessary to create here this button as we already have a `button` instance that we can re-use, created in the beforeEach block.

Reply:
Fixed, ty for help.